domains/DataCollection/FormGallery/index.js

```diff
@@ -52,9 +52,23 @@ const FormGallery = (props) => {
           ))}
         </ScrollView>
       </View>
+      {customForms.length < 1 && (
+        <View style={layout.screenRow}>
+          <Card>
+            <Card.Title title="You do not have any custom forms" />
+            <Card.Content>
+              <Text>Check out our marketplace to view available forms</Text>
+              <Button>View Marketplace</Button>
+            </Card.Content>
```

Comment:
I like how this looks! Comment it out for now so that we can use it when the marketplace is launched